### DISCUSSION TITLE
Getting all dates between a range

### DISCUSSION THREAD

**Discussion by cojoclaudiu** [js-fns/date-fns#2933] [Q&A]
Hello, it is possible to get all days between a range of dates?

<img width="496" alt="Screenshot 2022-01-27 at 22 06 52" src="https://user-images.githubusercontent.com/62800983/151451720-e4f94668-9011-49ff-a5be-e38b194e4a10.png">

For example I would like to include the last dates from first month which are not in range but are in between the range. I'm using a library (react-day-picker) in the example which is using the date-fns but the think is I would like to include / select the dates which are between a range.

**Comment by fturmel** [comment]
Have you tried `eachDayOfInterval`? https://date-fns.org/docs/eachDayOfInterval

```js
import { eachDayOfInterval } from "date-fns";

const start = new Date(2022, 0, 27);
const end = new Date(2022, 1, 3);
const res = eachDayOfInterval({ start, end });

console.log(res);
// 0: Thu Jan 27 2022 00:00:00 GMT-0500 (Eastern Standard Time)
// 1: Fri Jan 28 2022 00:00:00 GMT-0500 (Eastern Standard Time)
// 2: Sat Jan 29 2022 00:00:00 GMT-0500 (Eastern Standard Time)
// 3: Sun Jan 30 2022 00:00:00 GMT-0500 (Eastern Standard Time)
// 4: Mon Jan 31 2022 00:00:00 GMT-0500 (Eastern Standard Time)
// 5: Tue Feb 01 2022 00:00:00 GMT-0500 (Eastern Standard Time)
// 6: Wed Feb 02 2022 00:00:00 GMT-0500 (Eastern Standard Time)
// 7: Thu Feb 03 2022 00:00:00 GMT-0500 (Eastern Standard Time)
```

**Comment by cojoclaudiu** [comment]
@fturmel yes, but not the expected result. Here is my isolated code.
https://codesandbox.io/s/vibrant-smoke-29dch?file=/src/Calendar/index.tsx
